data/transactions/logic/eval.go

```diff
@@ -3081,6 +3081,40 @@ var ecdsaVerifyCosts = []int{
 	Secp256r1: 2500,
 }
 
+// ecdsaVerify checks a signature,
+func (cx *EvalContext) ecdsaVerify(curve EcdsaCurve, pkX, pkY []byte, msg []byte, sigR, sigS []byte) (result bool) {
+	// Go 1.19 panics on bad inputs. Catch it so that re can return false cleanly.
+	defer func() {
+		if recover() != nil {
```

Comment:
Leaving a note for when we return to the PR:  I'm open to discussion, but initial my preference is to catch the specific failure condition rather than the catch-all presented by the PR.

Additionally - I opened an issue to track upgrading to Go v1.19 with a checklist item referencing this PR:  https://github.com/algorand/go-algorand/issues/4759.  Intent is to confirm we finalize the PR prior to completing a v1.19 upgrade.